Dockerfile

```diff
@@ -7,5 +7,4 @@ RUN curl https://cli-assets.heroku.com/install.sh | sh
 WORKDIR /app
 RUN bundle config set force_ruby_platform true
 COPY Gemfile Gemfile.lock /app/
-RUN bundle install
```

Comment:
for context we were doing this here to reduce the heavy lifting on first run when using dev-containers - but since you're not using them, and curriculum doesn't do it that way anyway, I think this is a good call